components/ContactCard/ContactCard.tsx

```diff
@@ -13,67 +12,96 @@ import styles from './ContactCard.module.scss';
+                                <span className={styles.linkText}>{mobile}</span>
+                            </a>
+                        )}
+                    </div>
+                )}
+                {isSignature && (
+                    <div className={styles.linkGroup}>
+                        {email && (
+                            <a href={`mailto:${email}`} className={styles.link}>
+                                <span className={styles.linkText}>E. {email}</span>
+                            </a>
+                        )}
+                        {phone && (
+                            <a href={`tel:${phone}`} className={styles.link}>
+                                <span className={styles.linkText}>P. {phone}</span>
+                            </a>
+                        )}
+                        {mobile && (
+                            <a href={`tel:${mobile}`} className={styles.link}>
+                                <span className={styles.linkText}>M. {mobile}</span>
+                            </a>
+                        )}
+                        {website && (
+                            <a href={website.toString()} className={styles.link}>
+                                <span className={styles.linkText}>W. {website.hostname}</span>
+                            </a>
+                        )}
+                    </div>
+                )}
+                <div className={classNames(styles.linkGroup, styles.social)}>
+                    {website && isCard && (
+                        <a href={website.toString()} className={styles.link}>
                             <IconGlobe width={16} height={16} className={styles.icon} />
-                            <span className={styles.linkText}>{website}</span>
-                        </a>
-                    )}
-                    {mobile && (
-                        <a href={`tel:${mobile}`} className={styles.link}>
-                            <IconMobile width={16} height={16} className={styles.icon} />
-                            <span className={styles.linkText}>{mobile}</span>
-                        </a>
-                    )}
-                    {phone && (
-                        <a href={`tel:${phone}`} className={styles.link}>
-                            <IconPhone width={16} height={16} className={styles.icon} />
-                            <span className={styles.linkText}>{phone}</span>
                         </a>
                     )}
-                </div>
-                <div className={styles.linkGroup}>
                     {facebook && (
                         <a
                             href={`https://facebook.com/${facebook}`}
                             className={classNames(styles.link)}
                         >
                             <IconFacebook width={16} height={16} className={styles.icon} />
-                            <span className={styles.linkText}>{facebook}</span>
```

Comment:
Was removing the social text a design decision?
In either way, it would be nice to add a `title` to the links to mace it more accessible

Reply:
Yes, that's how it was designed. Not sure why the handles were omitted in this theme since we have them everywhere else in the new designs. Maybe we should keep them @iamhugofelix?

Reply:
I've added the titles for now, at least.

Reply:
Ok, here's the issue. I've followed the [design in Figma](https://www.figma.com/file/qGvtTUikFEzusn34lnaspD/Theme%3A-Lena?type=design&node-id=59%3A1460&t=L5zEg8USsoyTD0L9-1) (where the handles are not shown. As I already told @kudlajz I wanted to make as less changes to the cards as possible and that's the reason for not "making any changes".

TBH it is the same if we have the handles or not, it doesn't change it that much, so feel free to add them.